test/relocatable/plugins/checkPackages.py

```diff
@@ -0,0 +1,22 @@
+from bob.errors import ParseError
+
+# check if packages are relocatable or not
+
+def _checkPackages(step):
+    if step.isValid() and step.isPackageStep():
+        if not step.isRelocatable() and step.getPackage().getName() == 'some-tool-relocatable':
+            raise ParseError('Package some-tool-relocatable should be relocatable!')
+        if step.isRelocatable() and step.getPackage().getName() == 'some-tool':
```

Comment:
I think these kinds of tests should be unit-tests instead of black-box tests. Some black box tests still make sense to test the link/install logic.